docs/en/integrations/data-ingestion/clickpipes/index.md

```diff
@@ -152,9 +152,13 @@ Nullable versions of the above are also supported with these exceptions:
 
 ## List of Static IPs
 
-The following are the static IPs that ClickPipes uses to connect to your Kafka brokers:
+The following are the static IPs that ClickPipes uses to connect to your Kafka brokers separated by region. Add your related instance region to your IP Allowlist to allow traffic. Note that any region not listed will default to eu-central-1
```

Comment:
```suggestion
The following are the static NAT IPs that ClickPipes uses to connect to your Kafka brokers separated by region. Add your related instance region IPs to your IP allowlist to allow traffic. If your instance region is not listed here, it will fall to the default region: `eu-central-1` for EU regions and `us-east-2` for other regions.
```